Dockerfile

```diff
@@ -13,7 +13,8 @@ RUN mkdir /usr/src/policy-server
 WORKDIR /usr/src/policy-server
 COPY ./ ./
 
-RUN cargo install --target aarch64-unknown-linux-musl --path .
+RUN cargo install cargo-auditable
+RUN alias cargo="cargo auditable" && cargo install --target aarch64-unknown-linux-musl --path .
```

Comment:
Why not this?:

```suggestion
RUN cargo install cargo-auditable
RUN cargo auditable install --target aarch64-unknown-linux-musl --path .
```

Reply:
I didn't expect that to work, based on some GH issue I found, but it actually works!

I've changed the code as you suggested